package.json

```diff
@@ -43,25 +43,12 @@
     "@bentley/ui-abstract": "2.19.11",
     "@bentley/ui-core": "2.19.11",
     "@bentley/webgl-compatibility": "2.19.11",
-    "@svgr/cli": "^5.5.0",
-    "@swc/core": "^1.2.84",
-    "@swc/wasm": "^1.2.84",
-    "bufferutil": "^4.0.3",
-    "canvas": "^2.8.0",
-    "classnames": "^2.2.6",
-    "cpx": "^1.5.0",
-    "eslint": "^7.31.0",
-    "eslint-plugin-react-hooks": "^4.2.0",
-    "fibers": "^5.0.0",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.0",
-    "redux": "^4.1.1",
-    "type-fest": "^0.13.1",
-    "utf-8-validate": "^5.0.5"
+    "redux": "^4.1.1"
```

Comment:
@calebmshafer I notice that you removed redux from mobile-ui-react, and you are adding it here. Does mobile-sdk-core use redux?

Reply:
Oops, I misread; you aren't adding it, but leaving it. The question remains: is it needed?

Reply:
Good question, I don't think it does.  We can probably remove redux and react-redux.

Reply:
I think all react packages should be removed from here.

Reply:
Plus the only reason I left the React dependency is due to the ui-core dependency which I'd definitely want to re-visit when moving to 3.0.

The only use is UiEvent which I don't even understand why it exists over just the BeUiEvent in @bentley/bentleyjs-core.

Reply:
I'll approve this PR, and look at the rest.

Reply:
sounds good. Thanks!